tests/hello_world.cpp

```diff
@@ -3,12 +3,11 @@
 // University of Illinois/NCSA Open Source License.  Both these licenses can be
 // found in the LICENSE file.
 
-#include <stdio.h>
-
-class Test {}; // This will fail in C mode
+#include <iostream>
+#include <string>
 
 int main() {
-  printf("hello, world!\n");
+  std::cout << std::string("hello, world!\n");
   return 0;
 }
```

Comment:
This change worries me. Quite a lot of tests assume `hello_world.cpp` is just a plain `stdio`-using program. We have `hello_libcxx.cpp` for using iostreams.

Reply:
We have `hello_world.c` too for that purpose.    I didn't know about `hello_libcxx.cpp`.   Do you think we need 3 different flavors?

1. C with C stdio
2. C++ but with C stdio
3. C++ with C++ stdio

Maybe worth it.  I any case I'll remove this from this CL for sure and use `hello_libcxx.cpp`